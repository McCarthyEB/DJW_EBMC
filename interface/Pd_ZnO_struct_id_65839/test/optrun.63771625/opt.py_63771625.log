Running on machine: hawk
Running on host ccs0135
Time is Fri Oct 6 12:14:29 BST 2023
Launch directory is /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839
Working directory is /scratch/c.c1809139/63771625
Directory is /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839
Job ID is 63771625
Using ase script opt.py
Structure file is geometry.in
FHI-aims species path set to /home/scw1057/software/fhi-aims/species_defaults/defaults_2020/tight
FHI-aims Start Time is Fri Oct 6 12:14:29 BST 2023 running NCPUs=40 PPN=40
This jobs runs on the following machine: ccs0135
Number of Processing Elements is 40
Number of mpiprocs per node is 40

Start Time is Fri Oct 6 12:14:29 BST 2023

job = test
sub_dir = optrun.63771625
Will be working in directory /scratch/c.c1809139/63771625/test/optrun.63771625
results to go to directory /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/optrun.63771625
Creating working directory /scratch/c.c1809139/63771625/test/optrun.63771625
Creating results directory /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/optrun.63771625
Copying structure file /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/geometry.in to /scratch/c.c1809139/63771625/test/optrun.63771625
/scratch/c.c1809139/63771625/test/optrun.63771625 Running python script opt.py

With command:
python3 opt.py /scratch/c.c1809139/63771625/test/optrun.63771625 63771625 /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839 test optrun.63771625 hawk NoFix
fhi_aims run using ase script opt.py for job test running.
copying back for :  /scratch/c.c1809139/63771625/test/optrun.63771625
to home sub dir  :  /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/optrun.63771625
