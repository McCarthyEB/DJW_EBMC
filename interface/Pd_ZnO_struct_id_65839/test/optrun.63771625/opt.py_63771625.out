
real	0m5.672s
user	0m0.157s
sys	0m0.070s
Arguements passed 10
Argument       0: opt.py
Argument       1: /scratch/c.c1809139/63771625/test/optrun.63771625
Argument       2: 40
Argument       3: 63771625
Argument       4: /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839
Argument       5: test
Argument       6: optrun.63771625
Argument       7: hawk
Argument       8: NoFix
Argument       9: /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/hessian.aims
------------------------------------------------------------
Work directory                 : /scratch/c.c1809139/63771625/test/optrun.63771625
stdout file for prog           : /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/vasp_test_63771625.stdout
results_dir for this script    : /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/optrun.63771625
trajectory file for this run   : /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/optrun.63771625/test_63771625.traj
Stem for POSCAR files of steps : /home/c.c1809139/PhD/Oct_2023/Pd_ZnO_struct_id_65839/test/optrun.63771625/test_63771625.traj
Machine running job            : hawk

zfix not set so will not freeze any atom co-ordinates unless pre-defined in geometry file.
------------------------------------------------------------
Running command set to:
time mpirun -np 40 /apps/local/projects/scw1057/software/fhi-aims/bin/aims.230612.scalapack.mpi.x
Cell: Cell([[16.3029, 0.0, 0.0], [-8.151449999999997, 14.118725555357326, 0.0], [0.0, 0.0, 26.2229]])
0 O  -6.521078  13.177430   0.996732
1 O   1.630372   4.706195   0.996732
2 O   0.000082   7.529940   0.996732
3 O  -3.260498  13.177430   0.996732
4 O   6.521242   1.882450   0.996732
5 O   4.890952   4.706195   0.996732
6 O   3.260662   7.529940   0.996732
7 O   1.630372  10.353685   0.996732
8 O   0.000082  13.177430   0.996732
9 O   9.781822   1.882450   0.996732
10 O   6.521242   7.529940   0.996732
11 O   4.890952  10.353685   0.996732
12 O   3.260662  13.177430   0.996732
13 O  13.042402   1.882450   0.996732
14 O  11.412112   4.706195   0.996732
15 O   9.781822   7.529940   0.996732
16 O   8.151532  10.353685   0.996732
17 O   6.521242  13.177430   0.996732
18 Zn   4.890788   6.588786   1.625558
19 Zn   3.260498   3.765041   1.625558
20 Zn   3.260498   9.412531   1.625558
21 Zn  -3.260662   9.412531   1.625558
22 Zn   1.630208  12.236276   1.625558
23 Zn   1.630208   6.588786   1.625558
24 Zn  11.411948   0.941295   1.625558
25 Zn   1.630208   0.941295   1.625558
26 Zn   9.781658   3.765041   1.625558
27 Zn  -0.000082   9.412531   1.625558
28 Zn   8.151368   6.588786   1.625558
29 Zn  -4.890952  12.236276   1.625558
30 Zn   6.521078   9.412531   1.625558
31 Zn  -1.630372  12.236276   1.625558
32 Zn   4.890788  12.236276   1.625558
33 Zn  -1.630372   6.588786   1.625558
34 Zn  14.672528   0.941295   1.625558
35 Zn   8.151368   0.941295   1.625558
36 Zn  13.042238   3.765041   1.625558
37 Zn   4.890788   0.941295   1.625558
38 Zn  11.411948   6.588786   1.625558
39 Zn   6.521078   3.765041   1.625558
40 Zn   9.781658   9.412531   1.625558
41 Zn  -0.000082   3.765041   1.625558
42 Zn   8.151368  12.236276   1.625558
43 O   9.781658   3.765041   3.619022
44 O   1.630208   6.588786   3.619022
45 O   6.521078   3.765041   3.619022
46 O   8.151368   6.588786   3.619022
47 O  -3.260662   9.412531   3.619022
48 O   4.890788   0.941295   3.619022
49 O   6.521078   9.412531   3.619022
50 O   4.890788   6.588786   3.619022
51 O  -0.000082   9.412531   3.619022
52 O   4.890788  12.236276   3.619022
53 O  -1.630372   6.588786   3.619022
54 O   3.260498   9.412531   3.619022
55 O  14.672528   0.941295   3.619022
56 O   1.630208   0.941295   3.619022
57 O  -1.630372  12.236276   3.619022
58 O  13.042238   3.765041   3.619022
59 O   1.630208  12.236276   3.619022
60 O   3.260498   3.765041   3.619022
61 O  11.411948   6.588786   3.619022
62 O  -4.890952  12.236276   3.619022
63 O  11.411948   0.941295   3.619022
64 O   9.781658   9.412531   3.619022
65 O   8.151368   0.941295   3.619022
66 O  -0.000082   3.765041   3.619022
67 O   8.151368  12.236276   3.619022
68 Zn   4.890952  10.353685   4.247848
69 Zn   6.521242   1.882450   4.247848
70 Zn  -6.521078  13.177430   4.247848
71 Zn   0.000082  13.177430   4.247848
72 Zn   3.260662  13.177430   4.247848
73 Zn  -4.890788  10.353685   4.247848
74 Zn  -1.630208  10.353685   4.247848
75 Zn   4.890952   4.706195   4.247848
76 Zn  13.042402   1.882450   4.247848
77 Zn   9.781822   1.882450   4.247848
78 Zn   1.630372   4.706195   4.247848
79 Zn  -1.630208   4.706195   4.247848
80 Zn  11.412112   4.706195   4.247848
81 Zn   0.000082   1.882450   4.247848
82 Zn   8.151532   4.706195   4.247848
83 Zn   3.260662   7.529940   4.247848
84 Zn   9.781822   7.529940   4.247848
85 Zn  -3.260498  13.177430   4.247848
86 Zn   3.260662   1.882450   4.247848
87 Zn   6.521242   7.529940   4.247848
88 Zn   8.151532  10.353685   4.247848
89 Zn   0.000082   7.529940   4.247848
90 Zn   1.630372  10.353685   4.247848
91 Zn  -3.260498   7.529940   4.247848
92 Zn   6.521242  13.177430   4.247848
93 O   3.260662   7.529940   6.241312
94 O   1.630372   4.706195   6.241312
95 O  -4.890788  10.353685   6.241312
96 O   6.521242   1.882450   6.241312
97 O  13.042402   1.882450   6.241312
98 O   8.151532   4.706195   6.241312
99 O  -1.630208  10.353685   6.241312
100 O   1.630372  10.353685   6.241312
101 O   3.260662   1.882450   6.241312
102 O  11.412112   4.706195   6.241312
103 O  -6.521078  13.177430   6.241312
104 O   6.521242   7.529940   6.241312
105 O  -1.630208   4.706195   6.241312
106 O   4.890952   4.706195   6.241312
107 O   9.781822   7.529940   6.241312
108 O   0.000082  13.177430   6.241312
109 O   0.000082   7.529940   6.241312
110 O   4.890952  10.353685   6.241312
111 O  -3.260498  13.177430   6.241312
112 O   8.151532  10.353685   6.241312
113 O  -3.260498   7.529940   6.241312
114 O   0.000082   1.882450   6.241312
115 O   9.781822   1.882450   6.241312
116 O   3.260662  13.177430   6.241312
117 O   6.521242  13.177430   6.241312
118 Zn  14.672528   0.941295   6.870138
119 Zn   1.630208   0.941295   6.870138
120 Zn   1.630208  12.236276   6.870138
121 Zn   8.151368   6.588786   6.870138
122 Zn  -0.000082   9.412531   6.870138
123 Zn  -3.260662   9.412531   6.870138
124 Zn  13.042238   3.765041   6.870138
125 Zn   4.890788   6.588786   6.870138
126 Zn   8.151368   0.941295   6.870138
127 Zn  -1.630372   6.588786   6.870138
128 Zn   6.521078   9.412531   6.870138
129 Zn  11.411948   0.941295   6.870138
130 Zn  11.411948   6.588786   6.870138
131 Zn   1.630208   6.588786   6.870138
132 Zn  -4.890952  12.236276   6.870138
133 Zn   3.260498   3.765041   6.870138
134 Zn   3.260498   9.412531   6.870138
135 Zn   4.890788  12.236276   6.870138
136 Zn   9.781658   9.412531   6.870138
137 Zn  -1.630372  12.236276   6.870138
138 Zn   9.781658   3.765041   6.870138
139 Zn   6.521078   3.765041   6.870138
140 Zn   4.890788   0.941295   6.870138
141 Zn  -0.000082   3.765041   6.870138
142 Zn   8.151368  12.236276   6.870138
143 O  11.411948   0.941295   8.863602
144 O   6.521078   3.765041   8.863602
145 O   3.260498   9.412531   8.863602
146 O  13.042238   3.765041   8.863602
147 O   6.521078   9.412531   8.863602
148 O  -0.000082   9.412531   8.863602
149 O   4.890788   0.941295   8.863602
150 O   1.630208   6.588786   8.863602
151 O   3.260498   3.765041   8.863602
152 O   9.781658   3.765041   8.863602
153 O  11.411948   6.588786   8.863602
154 O   8.151368   0.941295   8.863602
155 O   4.890788  12.236276   8.863602
156 O  -3.260662   9.412531   8.863602
157 O   1.630208  12.236276   8.863602
158 O   4.890788   6.588786   8.863602
159 O   1.630208   0.941295   8.863602
160 O   9.781658   9.412531   8.863602
161 O  -0.000082   3.765041   8.863602
162 O   8.151368   6.588786   8.863602
163 O  14.672528   0.941295   8.863602
164 O  -1.630372  12.236276   8.863602
165 O  -4.890952  12.236276   8.863602
166 O  -1.630372   6.588786   8.863602
167 O   8.151368  12.236276   8.863602
168 Zn  11.412112   4.706195   9.492428
169 Zn   8.151532   4.706195   9.492428
170 Zn   0.000082  13.177430   9.492428
171 Zn  -1.630208  10.353685   9.492428
172 Zn   0.000082   7.529940   9.492428
173 Zn  -1.630208   4.706195   9.492428
174 Zn   3.260662  13.177430   9.492428
175 Zn   4.890952   4.706195   9.492428
176 Zn   9.781822   7.529940   9.492428
177 Zn  -6.521078  13.177430   9.492428
178 Zn  -4.890788  10.353685   9.492428
179 Zn   6.521242   7.529940   9.492428
180 Zn   1.630372  10.353685   9.492428
181 Zn   9.781822   1.882450   9.492428
182 Zn   0.000082   1.882450   9.492428
183 Zn  13.042402   1.882450   9.492428
184 Zn   8.151532  10.353685   9.492428
185 Zn   6.521242   1.882450   9.492428
186 Zn   3.260662   1.882450   9.492428
187 Zn  -3.260498   7.529940   9.492428
188 Zn  -3.260498  13.177430   9.492428
189 Zn   4.890952  10.353685   9.492428
190 Zn   3.260662   7.529940   9.492428
191 Zn   1.630372   4.706195   9.492428
192 Zn   6.521242  13.177430   9.492428
193 O  -1.630208  10.353685  11.485892
194 O   3.260662  13.177430  11.485892
195 O   9.781822   1.882450  11.485892
196 O   6.521242   7.529940  11.485892
197 O   0.000082   1.882450  11.485892
198 O  -1.630208   4.706195  11.485892
199 O   9.781822   7.529940  11.485892
200 O   3.260662   7.529940  11.485892
201 O   3.260662   1.882450  11.485892
202 O  -4.890788  10.353685  11.485892
203 O  -6.521078  13.177430  11.485892
204 O  13.042402   1.882450  11.485892
205 O   0.000082  13.177430  11.485892
206 O   4.890952   4.706195  11.485892
207 O   4.890952  10.353685  11.485892
208 O   8.151532  10.353685  11.485892
209 O   8.151532   4.706195  11.485892
210 O   1.630372   4.706195  11.485892
211 O  -3.260498   7.529940  11.485892
212 O   6.521242   1.882450  11.485892
213 O   0.000082   7.529940  11.485892
214 O  11.412112   4.706195  11.485892
215 O   1.630372  10.353685  11.485892
216 O  -3.260498  13.177430  11.485892
217 O   6.521242  13.177430  11.485892
218 Zn  11.411948   6.588786  12.114718
219 Zn  -1.630372   6.588786  12.114718
220 Zn  14.672528   0.941295  12.114718
221 Zn   3.260498   9.412531  12.114718
222 Zn   3.260498   3.765041  12.114718
223 Zn  -3.260662   9.412531  12.114718
224 Zn  -0.000082   3.765041  12.114718
225 Zn   4.890788   6.588786  12.114718
226 Zn   9.781658   9.412531  12.114718
227 Zn  11.411948   0.941295  12.114718
228 Zn   4.890788   0.941295  12.114718
229 Zn  13.042238   3.765041  12.114718
230 Zn   8.151368   6.588786  12.114718
231 Zn   4.890788  12.236276  12.114718
232 Zn   6.521078   3.765041  12.114718
233 Zn  -0.000082   9.412531  12.114718
234 Zn   8.151368   0.941295  12.114718
235 Zn   8.151368  12.236276  12.114718
236 Pd   9.772855   7.509327  16.635021
237 Pd  12.221714   6.095477  16.635021
238 Pd  -1.632083   4.681487  16.635021
239 Pd   0.816938   3.267638  16.635021
240 Pd   3.265797   1.853789  16.635021
241 Pd   5.714900   0.439798  16.635021
242 Pd   0.012472  13.144675  16.635021
243 Pd   2.461330  11.730826  16.635021
244 Pd   4.910352  10.316976  16.635021
245 Pd   7.359455   8.902986  16.635021
You are using i-Pi based socket connectivity between ASE and FHI-aims.
The communicated energy in Hartree units will be converted to eV in ASE and not FHI-aims.
The eV/Hartree unit in FHI-aims is given by CODATA 2002 (Web Version 4.0 2003-12-09), Peter J. Mohr, Barry N. Taylor
ASE uses CODATA 2014, thus the energy in eV from ASE and the FHI-aims outputs will differ.
Please be consistent in the unit conversion for data analysis!
You can turn off this message by setting 'codata_warning' keyword to False.
Traceback (most recent call last):
  File "/apps/chemistry/ase/3.20.1/el7/AVX512/gnu-7.3/ase/calculators/socketio.py", line 291, in _accept
    self.clientsocket, self.address = self.serversocket.accept()
  File "/apps/languages/python/intel/2020u1/intelpython3/lib/python3.7/socket.py", line 212, in accept
    fd, addr = self._accept()
socket.timeout: timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "opt.py", line 216, in <module>
    energy=atoms.get_potential_energy()
  File "/apps/chemistry/ase/3.20.1/el7/AVX512/gnu-7.3/ase/atoms.py", line 733, in get_potential_energy
    energy = self._calc.get_potential_energy(self)
  File "/apps/chemistry/ase/3.20.1/el7/AVX512/gnu-7.3/ase/calculators/calculator.py", line 682, in get_potential_energy
    energy = self.get_property('energy', atoms)
  File "/apps/chemistry/ase/3.20.1/el7/AVX512/gnu-7.3/ase/calculators/calculator.py", line 738, in get_property
    self.calculate(atoms, [name], system_changes)
  File "/apps/chemistry/ase/3.20.1/el7/AVX512/gnu-7.3/ase/calculators/socketio.py", line 637, in calculate
    results = self.server.calculate(atoms)
  File "/apps/chemistry/ase/3.20.1/el7/AVX512/gnu-7.3/ase/calculators/socketio.py", line 352, in calculate
    self._accept()
  File "/apps/chemistry/ase/3.20.1/el7/AVX512/gnu-7.3/ase/calculators/socketio.py", line 297, in _accept
    ' with status {}'.format(status))
OSError: Subprocess terminated unexpectedly with status 0
